.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+- '3.5'
+- '3.6'
+before_install:
+- pip install -U pip setuptools pytest
+install:
+- python setup.py install
+script:
+- python setup.py test --addopts -v
```

Comment:
Also consider setting up `tox` and using [tox-travis](https://pypi.python.org/pypi/tox-travis) here.